Australian Digital Observatory/models/event.ini

```diff
@@ -0,0 +1,37 @@
+[model]
+name = Event
+label = {{ this.title }}
+hidden = yes
+
+[fields.title]
+label = Title
+type = string
+size = large
+
+[fields.event_date]
+label = Event Date
+type = datetime
+
+[fields.where]
+label = Where will the event be held?
+type = string
+
+[fields.type]
+label = Type of event
+type = select
+choices = workshop, conference, talk
+choice_labels = Workshop, Conference, Talk
+
+
+[fields.organiser]
+label = Event Organiser
+type = string
+width = 1/2
+
+[fields.description]
+label = Description
+type = markdown
+
+[fields.registration]
```

Comment:
Would it be more generic/flexible if we called this field 'URL' instead of 'Registration Link'? Only reason I suggest this is because sometimes we might have a more generic URL to the event, which then has a registration link on it. For example, ResBaz usually has an awesome website that we want to link to, which we wouldn't be sharing if we only included the registration link (which from memory was just an EventBrite link). Alternatively, we could have two URL fields?

Reply:
That's a good point. I'll change it